plugin/pkg/scheduler/scheduler.go

```diff
@@ -236,6 +233,9 @@ func (sched *Scheduler) scheduleOne() {
 			})
 			return
 		}
+		if err := sched.config.SchedulerCache.FinishBinding(&assumed); err != nil {
```

Comment:
Sorry for the delay :). We should delete the assumedPod if binding failed; the assumed pod was added into cache and account into "PodFitResource" predicates, there'll be resources "lost" if failed binding pod not deleted.